rust/abacus-base/src/contract_sync/interchain_gas.rs

```diff
@@ -73,6 +73,10 @@ where
                 for gas_payment in gas_payments.iter() {
                     db.process_gas_payment(gas_payment)?;
                 }
+                // TODO: this is technically not recording the number of events in the DB but rather
```

Comment:
Is this actually a TODO?

Reply:
I guess we should ask if we care. I use the TODO more as a "we should at least talk about it"

Reply:
Nit: I think something we should talk about, or a future improvement should be put in a ticket, code comments should just explain what the code does as of right now

Reply:
I will make a ticket. This logic makes more sense.